testgrid/cmd/entomologist/main.go

```diff
@@ -211,54 +266,91 @@ func getIssues(client githubClient, o options) ([]github.Issue, error) {
 	return issues, err
 }
 
-// Algorithm for detecting and formatting test targets in issues
-func pinIssues(client githubClient, o options) (*issue_state.IssueState, error) {
-	issues, err := getIssues(client, o)
+// matchesPattern determines what Entomologist sees as a potential association to a test group
+var targetRegExp = regexp.MustCompile(`(?m)^target:(.+)$`)
```

Comment:
Discussed offline, 'Pin:' or 'Pin to' or similar might be more clear here since now it's matching test groups.

Also, TODO to make this match on tabs?